config/reader.go

```diff
@@ -506,16 +406,9 @@ func readFromCloud(
 		logger.Debug("reading tlsCertificate from the cloud")
 		// Use the SignalingInsecure from the Cloud config returned from the app not the initial config.
 
-		var certData *Cloud
-		if originalCfg.Cloud.AppAddress == "" {
-			certData, err = readCertificateDataFromCloud(ctx, cfg.Cloud.SignalingInsecure, cloudCfg)
-		} else {
-			certData, err = readCertificateDataFromCloudGRPC(ctx, cfg.Cloud.SignalingInsecure, cloudCfg, logger)
-		}
-
+		certData, err := readCertificateDataFromCloudGRPC(ctx, cfg.Cloud.SignalingInsecure, cloudCfg, logger)
 		if err != nil {
-			var urlErr *url.Error
```

Comment:
gRPC will never return a url.Error so the check `(!errors.As(err, &urlErr) || urlErr.Temporary())` is meaningless. To avoid the test now failing i disabled cert reading on the calls from the test.